docs/how_jstz_works.md

```diff
@@ -0,0 +1,60 @@
+---
+title: 👨‍⚖️ How jstz works?
+---
+
+<script setup>
+import VPButton from "vitepress/dist/client/theme-default/components/VPButton.vue";
+</script>
+
+# 👨‍⚖️ `How jstz works?`
+
+## Overview
+
+`jstz` is a JavaScript/Typescript server runtime for Tezos 2.0 designed to provide a great developer experience by aiming to be compatible with web conventions.
+
+Through `jstz` developers can set up, deploy and test so called _smart functions_ written in Javascript/Typescript that can get directly executed on Tezos L2.
```

Comment:
executed on Tezos L2 -> executed on the `jstz` smart rollup node. 
Link https://tezos.gitlab.io/alpha/smart_rollups.html on smart rollup

Reply:
Please move the running of smart functions using HTTP requests to this paragraph.